src/pages/postgraphile/smart-tags-file.md

```diff
@@ -65,7 +65,7 @@ extension must be `.json5`) and is formatted like this:
          * We've added a shortcut to class-types so you can tag/describe
          * columns at the same time of the class.
          */
-        columns: {
+        attributes: {
```

Comment:
It's 'attribute' rather than 'attributes' (it comes from the postgres table name: `pg_attribute`)

```suggestion
        attribute: {
```